### PR TITLE
Added Accessibility Question

### PR DESCRIPTION
**Pull request by preeformance**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Adds to #374 

Hey, so glad to see this as a new quiz category! I'm happy to add more questions but want to do things in the way that's easiest for the maintainers to keep up with. Is one question per PR the way to go or would multiple at once be a better option?

### CODE REVIEW

**Comment by jdwilkin4** [comment]
Hi @preeformance ! 

Either option works fine.
If you have a list of questions right now, then you can add all of that to the same PR.
When that is merged and you think of new questions then you can create a new PR for those 👍

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const accessibilityQuiz = [
+   {
+    Question: "What attribute makes images accessible for people using screen readers?",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:3]:
```suggestion
    Question: "Which attribute makes images accessible for people using screen readers?",
```

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    Answer: "alt",
+    Distractor1: "desc",
+    Distractor2: "img",
+    Distractor3: "src",
+    Explanation:
+      "With one attribute, alt, we are able to make the image accessible for screen reader users.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:9]:
```suggestion
      "The alt attribute provides short descriptive text for images and the text is read aloud by a screen reader.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by preeformance** [comment]
I'll create a new PR after I write a few more questions. Thanks!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**